### PR TITLE
Mobile Mixpanel setup

### PR DESCRIPTION
**Pull request by tbezman**
Same pattern as the `ErrorReportingContext`. Just import `useTrack` and you're golden

### CODE REVIEW

File: apps/mobile/src/contexts/MobileAnalyticsProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import AnalyticsProvider, {
+  IdentifyFunction,
+  TrackFunction,
+} from '~/shared/contexts/AnalyticsContext';
+
+let instance: Mixpanel | undefined = undefined;
+
+// TODO env
+const token = '39358d4e4b8aeb0ee10e4ddbcb7a53eb';
+const apiUrl = 'https://analytics.gallery.so';
```
Comment (Robinnnnn) [inline apps/mobile/src/contexts/MobileAnalyticsProvider.tsx:13]: did you mean to handle this TODO in a later PR?

**Review by Robinnnnn — APPROVED** [review]
love the shared context!

**Comment by kaitoo1** [comment]
BIG

File: apps/mobile/src/contexts/MobileAnalyticsProvider.tsx (hunk 1 above)
Comment (tbezman) [reply apps/mobile/src/contexts/MobileAnalyticsProvider.tsx:13]: yeah I added some more of these around the codebase. just so I have something to grep when we finally support multiple environments

**Pull request merged**